.circleci/config.yml

```diff
@@ -924,6 +924,14 @@ workflows:
 #        name: test/e2e-ipv6
 #        # custom parameters
 #        ipv6: true
+    - e2e:
+        <<: *master_workflow_filters
+        name: test/e2e V2
+        requires:
+          - images
+          - check
+        # custom parameters
+        api: v2
```

Comment:
We already have this in `kuma-master` workflow. Wasn't this executed?

Reply:
I wanted to make this pass in the PR, will remove it before merging. Should have been more clear in the patch

Reply:
and I added it with the wrong filter